util/get_vulns.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'node-fetch';
+const snykApi = 'http://snyk-widget.herokuapp.com';
+const key = 'VRHr?YP_Gb7d9LFh$ugktZHUe+gGBp#LGG3*bDLt';
```

Comment:
Could these be moved into `data/config.js`?

Reply:
Yes that would be better :)